streams/src/main/java/org/apache/kafka/streams/internals/StreamsConfigUtils.java

```diff
@@ -28,11 +28,11 @@ public class StreamsConfigUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(StreamsConfigUtils.class);
 
-    @SuppressWarnings("deprecation")
     public enum ProcessingMode {
         AT_LEAST_ONCE(StreamsConfig.AT_LEAST_ONCE),
 
-        EXACTLY_ONCE_ALPHA(StreamsConfig.EXACTLY_ONCE),
+        // TODO cleanup
+        EXACTLY_ONCE_ALPHA("exactly_once"),
```

Comment:
Will be addressed in a follow up refactoring PR. Just a workaround to keep the scope contained.